tools/project_manager/show_word_context.js

```diff
@@ -1,17 +1,18 @@
 /*global splitControl pageBrowse showWordContext proofIntData */
 
-window.addEventListener("DOMContentLoaded", function() {
+window.addEventListener("DOMContentLoaded", function () {
     let storageKeyLayout = showWordContext.storageKey + "-layout";
     let layout;
     try {
         layout = JSON.parse(localStorage.getItem(storageKeyLayout));
-    } catch(error) { // eslint-disable-line no-unused-vars
+        // eslint-disable-next-line no-unused-vars
```

Comment:
I'll actually just fix the eslint error. I previously changed it from disable line to disable next line since the formatter moved the comment above the unused error variable.